cypress/integration/app_test.js

```diff
@@ -101,6 +103,20 @@ describe("Tests",function(){
     // .contains('Some username')
     // This is not work some error is happend!!!
 
+    .get('input:first').type('Some username').type('{enter}')
+    .get('ul li:last').click().url().should('eq', 'http://localhost:3000/login')
 
   })
+  it("Making more bad actions",function(){
+    cy
+    .visit('http://localhost:3000/signup')
+    .get('form').submit()
+    .find('span').should('have.class', 'help-block')
+    .and('contain', 'This field')
+    .reload()
+    .get('form').signUp()
+  })
+  it("XHR at login page",function(){
```

Comment:
Ovaj moze napisati ovako
```
it("XHR at login page");
```
it bez funkcije kao drugog parametra, racunace ga kao da je `work in progress` test